### ISSUE TITLE
No implemented error at pod install

### ISSUE THREAD

**Issue by carlosetorresm** [ukieTux/media_scanner#2]
It is specified that the plugin only works on android but when working on a Ios/Android app it should run

```
[!] No podspec found for `media_scanner` in `.symlinks[/plugins/media_scanner/ios]()`
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/external_sources/path_source.rb:14]():in `block in fetch'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/user_interface.rb:64]():in `section'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer/analyzer.rb:970]():in `fetch_external_sources'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer/analyzer.rb:117]():in `analyze'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer.rb:416]():in `analyze'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer.rb:241]():in `block in resolve_dependencies'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/user_interface.rb:64]():in `section'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer.rb:240]():in `resolve_dependencies'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/installer.rb:161]():in `install!'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/command/install.rb:52]():in `run'
    [/Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334]():in `run'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/lib/cocoapods/command.rb:52]():in `run'
    [/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.11.2/bin/pod:55]():in `<top (required)>'
    [/usr/local/bin/pod:23]():in `load'
    [/usr/local/bin/pod:23]():in `<main>'
```

**Comment by XuanTung95** [comment]
Same error. Any solution?

**Comment by XuanTung95** [comment]
I created #4 to remove ios plugin

**Comment by Lzyct** [comment]
This problem was solved in version 2.1.0
Thanks
